### ISSUE TITLE
Is there a way to control the scrolling of a pdf?

### ISSUE THREAD

**Issue by alanscott** [barteksc/AndroidPdfViewer#913]
I am trying to work out if there is a way to control the scrolling of a pdf, for instance, have a downward pointing arrow button, that when I press it, it will scroll the pdf a certain amount. From what I can gather the only thing relating to this is the scroll handler but I don't see anything that you could attach to a button. Is there a means of doing this? thank you

**Comment by 1stmetro** [comment]
The library does have the option of a tab which shows page numbers and you scroll by dragging the tab, i presume you can assign a custom view and use your own icon, not played around with this code,

**Comment by alanscott** [comment]
Yes, I have been able to work with that, but it's not quite what I am looking for. More the ability to take control of the scrolling so I can assign it how I like. Eventually I wish to use computer vision to control scrolling of a pdf through gesture recognition. Thank you for your response.

**Comment by 1stmetro** [comment]
From memory you have the scrollTo() and scrollBy() if this helps ?

**Comment by alanscott** [comment]
Yes! This is it..So simple but I could not see it.. Thank You!
